static/app/utils/theme/theme.tsx

```diff
@@ -802,62 +803,90 @@ const iconDirectionToAngle: Record<IconDirection, number> = {
 
 export type FormSize = 'xs' | 'sm' | 'md';
 
-type FormSizes = {
-  [key in FormSize]: {
-    fontSize: string;
-    height: number;
-    lineHeight: string;
-    minHeight: number;
+export type FormTheme = {
+  form: {
+    [key in FormSize]: {
+      fontSize: string;
+      height: number;
+      lineHeight: string;
+      minHeight: number;
+    };
+  };
+  formPadding: {
+    [key in FormSize]: {
+      paddingBottom: number;
+      paddingLeft: number;
+      paddingRight: number;
+      paddingTop: number;
+    };
+  };
+  formRadius: {
+    [key in FormSize]: {
+      borderRadius: string;
+    };
   };
 };
 
-const formSizes: FormSizes = {
-  md: {
-    height: 38,
-    minHeight: 38,
-    fontSize: '0.875rem',
-    lineHeight: '1rem',
-  },
-  sm: {
-    height: 32,
-    minHeight: 32,
-    fontSize: '0.875rem',
-    lineHeight: '1rem',
-  },
-  xs: {
-    height: 26,
-    minHeight: 26,
-    fontSize: '0.75rem',
-    lineHeight: '0.875rem',
+const formTheme: FormTheme = {
+  /**
+   * Common styles for form inputs & buttons, separated by size.
+   * Should be used to ensure consistent sizing among form elements.
+   */
+  form: {
+    md: {
+      height: 38,
+      minHeight: 38,
+      fontSize: '0.875rem',
+      lineHeight: '1rem',
+    },
+    sm: {
+      height: 32,
+      minHeight: 32,
+      fontSize: '0.875rem',
+      lineHeight: '1rem',
+    },
+    xs: {
+      height: 26,
+      minHeight: 26,
+      fontSize: '0.75rem',
+      lineHeight: '0.875rem',
+    },
   },
-} as const;
 
-type FormPaddingSizes = {
-  [key in FormSize]: {
-    paddingBottom: number;
-    paddingLeft: number;
-    paddingRight: number;
-    paddingTop: number;
-  };
-};
-const formPaddingSizes: FormPaddingSizes = {
-  md: {
-    paddingLeft: 16,
-    paddingRight: 12,
-    paddingTop: 10,
-    paddingBottom: 10,
-  },
-  sm: {
-    paddingLeft: 12,
-    paddingRight: 10,
-    paddingTop: 8,
-    paddingBottom: 8,
+  /**
+   * Padding for form inputs
+   * @TODO(jonasbadalic) This should exist on form component
+   */
+  formPadding: {
+    md: {
+      paddingLeft: 16,
+      paddingRight: 12,
+      paddingTop: 10,
+      paddingBottom: 10,
+    },
+    sm: {
+      paddingLeft: 12,
+      paddingRight: 10,
+      paddingTop: 8,
+      paddingBottom: 8,
+    },
+    xs: {
+      paddingLeft: 8,
+      paddingRight: 6,
+      paddingTop: 6,
+      paddingBottom: 6,
+    },
   },
-  xs: {
-    paddingLeft: 8,
-    paddingRight: 6,
-    paddingTop: 6,
-    paddingBottom: 6,
+  formRadius: {
+    md: {
+      borderRadius: '6px',
+    },
+    sm: {
+      borderRadius: '6px',
+    },
+    xs: {
+      borderRadius: '6px',
+    },
```

Comment:
Curious why these are `string` but padding is `number`?

Reply:
yeah, good question. I basically took this from the existing `theme.borderRadius` value:

https://github.com/getsentry/sentry/blob/f742e59be7571e6ffc370150583b88cdb64aabf7/static/app/utils/theme/theme.tsx#L1011

for the old theme, we need them to be the same value, as we didn’t change the border-radius for sizes. For `chonk`, we want them to be adaptive.

`theme.borderRadius` should eventually be removed.

One thing why I think we should have strings with `px` over numbers is that if we use them in `calc` expressions, they won’t work without the `px` suffix. I think emotion just replaces `border-radius: 6` with `6px`, but inside `calc`, it won’t.

Probably we’d want `padding` to be a `string` then, too ?